config/shredder.batch.config.reference.hocon

```diff
@@ -68,6 +68,20 @@
     ]
   },
 
+  # Specifies interval shredder will work on
+  "runInterval": {
+    # Timestamp specifies since when shredder will process
+    "sinceTimestamp": "2021-10-12-14-55-22",
```

Comment:
Just pointing out this is an unusual format for a date-time.  I suppose it's consistent with folder names, so I think you can justify the decision.  I just wanted to mention it.

Does the shredder process folders **after** this date or **after and including** this date?  Maybe mention it in the description.

And mention that it's optional?

Reply:
Exactly, the reason was making the format same with folder names.

Processing starts after this date. I will include in the description. Also, I will mention it is optional 👍